From 1d0fa519ef017aef4821af49c59bad119e7f0d66 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 26 Nov 2024 23:23:42 -0800
Subject: [PATCH] autogen.sh: remove '-unknown' in version

python setuptools >=66.0.0 treats '-unknown' as an invalid version.
The error message is as below:
  pkg_resources.extern.packaging.version.InvalidVersion: Invalid version: '1.18.0-unknown'

Remove the '-unknown' suffix to fix this issue.

Upstream-Status: Submitted [https://lists.gnupg.org/pipermail/gnupg-devel/2023-February/035293.html]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>

Rebase to 1.24.0
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 9f91297..9f7559b 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -271,7 +271,7 @@ if [ "$myhost" = "find-version" ]; then
     else
       ingit=no
       beta=yes
-      tmp="-unknown"
+      tmp=""
       cid="0000000"
       rev="0000000"
       rvd="0"
